### ISSUE TITLE
2.0.20123: GameSelectorModel#load:68 - NullPointerException

### ISSUE THREAD

**Issue by tripleabuilderbot** [triplea-game/triplea#6746]
## User Description
my downloaded maps don't show on my selected maps

## TripleA Version
2.0.20123

## Java Version
11.0.6

## Operating System
Windows 10

## Exception 
java.lang.NullPointerException

## Stack Trace
```
java.lang.NullPointerException
	at games.strategy.engine.framework.startup.ui.panels.main.game.selector.GameSelectorModel.load(GameSelectorModel.java:68)
	at games.strategy.engine.framework.startup.ui.panels.main.game.selector.GameSelectorModel.loadDefaultGameSameThread(GameSelectorModel.java:208)
	at games.strategy.engine.framework.GameRunner.loadGameIfSpecified(GameRunner.java:98)
	at games.strategy.engine.framework.GameRunner.showMainFrame(GameRunner.java:83)
	at java.base/java.lang.Thread.run(Thread.java:834)

```

**Comment by DanVanAtta** [comment]
A fix for this is in-flight, should be built and ready in about 30 minutes.

**Comment by DanVanAtta** [comment]
This should be fixed in 2.0.20183
